enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -48,24 +48,26 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
             </div>
             <div class="course-info">
               <i class="fa fa-clock-o" aria-hidden="true"></i>
-              <span>${ starts_at_text } ${ course_start_date } &nbsp;| &nbsp; ${ course_pacing } - ${ course_pace_text }</span>
+              <span>${ starts_at_text } ${ course_start_date } &nbsp;| &nbsp; ${ course_pacing }</span>
```

Comment:
@haikuginger That makes sense. I mistakenly thought it was introduced in the diff for this PR as well, but I see now that it wasn't